source/assets/stylesheets/all.scss

```diff
@@ -17,6 +17,22 @@
 @import "components/portfolio";
 @import "components/rubyshop";
 
+#events {
+  .timeline-image {
```

Comment:
Ah yes, I see what you mean. Good point.

Scoping it to `#events` is safer, but maybe it's premature "optimization".